ceremonies/src/lib.rs

```diff
@@ -1900,10 +1920,24 @@ impl<T: Config> Pallet<T> {
 		None
 	}
 
-	#[cfg(any(test, feature = "runtime-benchmarks"))]
+	pub fn validate_reputation(
+		account_id: &T::AccountId,
+		cid: &CommunityIdentifier,
+		cindex: CeremonyIndexType,
+	) -> bool {
+		let current_cindex = <encointer_scheduler::Pallet<T>>::current_ceremony_index();
+		if cindex < current_cindex.saturating_sub(Self::reputation_lifetime()) {
+			return false
+		}
+		<ParticipantReputation<T>>::get((*cid, cindex), account_id).is_verified()
+	}
+
+	#[cfg(any(test, feature = "runtime-benchmarks", feature = "mocks"))]
 	// only to be used by tests
-	fn fake_reputation(cidcindex: CommunityCeremony, account: &T::AccountId, rep: Reputation) {
+	pub fn fake_reputation(cidcindex: CommunityCeremony, account: &T::AccountId, rep: Reputation) {
```

Comment:
this looks helpful. we may even consider exposing this as a root origin extrinsic on testnets